app/js/components/orderEntry/Orders.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Accordion from './Accordion';
+import allOrdersMock from './allOrdersMock.json';
+import OrderHeader from './OrderHeader';
+import LabOrderDetails from './LabOrderDetails';
+import DrugOrderDetails from './DrugOrderDetails';
+
+const Orders = () => (
+  <React.Fragment>
+    {allOrdersMock.orders.map(order =>
```

Comment:
are we still gon use mock data still ?

Reply:
the data we'll be using is currently not available as we would need to fetch all the orders of the patient (drugs and lab).
It would be handled in another ticket.